RFC-0005-dyllib-cpp.md

```diff
@@ -0,0 +1,65 @@
+# **RFC-0005 for Presto Creating a Dynamically Linked Functions Library in CPP**
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on creating your RFC and the process surrounding it.
+
+## Creating a Dynamically Linked Functions Library in CPP
+
+Proposers
+
+* Soumya Duriseti
+* Tim Meehan
+
+## [Related Issues]
+
+https://github.com/facebookincubator/velox/pull/1005
+
+## Summary
+
+This proposed change expands the dynamic function loading ability to cpp user defined functions (UDFs). The (Velox?) worker is to access said code. The dynamic functions are to be loaded upon running an instance of the presto server. In the presto server instance, it will search for any .so or .dylib files and load them using this library.
+## Background
+
+Currently, on Presto, any Java UDFs can be loaded dynamically. This is an important feature for any client who wants their Presto build to be lightweight and to protect any non-public code or information from having to be in the open source space. Creating UDFs is an important functionality Presto has and its competitors offer as well. Being able to load CPP functions dynamically will extend the offering in Prestissimo/Velox(?).
+
+### [Optional] Goals
+
+### [Optional] Non-goals
+
+## Proposed Implementation
+The general process is as follows:
+
+1. What modules are involved
```

Comment:
Can you add some information on when the functions will be registered and what component will do the registering?  Just some code level information.